Toolset/palettes/project browser/revprojectbrowserbehavior.livecodescript

```diff
@@ -132,6 +132,21 @@ on preOpenStack
    ideSelectedObjectChanged
 end preOpenStack
 
+on doRefreshProjectView
+    local tStacks, tCards
+
+    lock screen
+    put the openstacks into tStacks
+    repeat for each line tStack in tStacks
+        put the cardIDs of stack tStack into tCards
+        repeat for each line tCardID in tCards
+            refreshChildren the long name of card id tCardID of stack tStack
```

Comment:
refreshChildren the long name of card id tCardID of stack tStack
should be 
refreshChildren the long id of card id tCardID of stack tStack

long name breaks e.g 
function ideStackOfObject pLongID
   local tOffset
   put wordOffset("stack",pLongID) into tOffset

if card name contains "Stacks in Use" as for the message box